BUILD.bazel

```diff
@@ -33,14 +33,19 @@ load("@bazel_gazelle//:def.bzl", "gazelle")
 # - A testdata "repo" that looks like a go package, but isn't
 #   (testdata/src/example.com)
 #
+# Excluding all SQLPARSER_TARGETS to avoid gazelle removing them
```

Comment:
[nit] I meant adding this as a bullet point to the list above.
```
# - A testdata "repo" that looks like a go package, but isn't
#   (testdata/src/example.com)
# - A few auto-generated sql parser files.
```

Reply:
Added